eternalcore-core/src/main/java/com/eternalcode/core/feature/teleport/Teleport.java

```diff
@@ -18,7 +18,9 @@ public class Teleport {
         this.playerUniqueId = playerUniqueId;
         this.startLocation = startLocation;
         this.destinationLocation = destinationLocation;
-        this.teleportMoment = Instant.now().plus(time);
+
+        // Adds 1 second so that the timer starts counting from 5 instead of 4, when the time in the config is set to 5.
+        this.teleportMoment = Instant.now().plus(time).plusSeconds(1);
```

Comment:
This is okey but you should do it while formatting duration#plus because you change real teleport time